src/ViewModel/Converter/ArticleDownloadLinksListConverter.php

```diff
@@ -31,11 +31,14 @@ public function convert($object, string $viewModel = null, array $context = [])
 
         $downloads = [];
         $types = [];
+
+        $articleUri = $this->urlGenerator->generate('article', [$object], UrlGeneratorInterface::ABSOLUTE_URL);
```

Comment:
can we rename this `canonicalUri` or similar? Or just add a comment that this is the canonical URI?

Reply:
Have renamed.